### ISSUE TITLE
Can't complete install due to index.php error

### ISSUE THREAD

**Issue by coreysyms** [GoogleCloudPlatform/php-docs-samples#699]
Following the instructions on this wordpress install repo I went to deploy via `gcloud app deploy app.yaml cron.yaml` This runs fine and tells me success and to go to my site via web address ` https://PROJECT_ID.appspot.com/`

It is on this page that I get this error.

```Warning: require(/srv/index.php): failed to open stream: No such file or directory in /srv/gae-app.php on line 53 Fatal error: require(): Failed opening required '/srv/index.php' (include_path='.:/usr/lib/php') in /srv/gae-app.php on line 53```

Here are the files I have in my dirs.

```
corey@cloudshell:~/php-docs-samples/appengine/php72/wordpress (wp-codelab-215015)$ ls
composer.json  composer.lock  files  my-wordpress-project  phpunit.xml.dist  README.md  test  vendor  wordpress.php
```

and 

```
corey@cloudshell:~/php-docs-samples/appengine/php72/wordpress/my-wordpress-project (wp-codelab-215015)$ ls
app.yaml  cron.yaml  gae-app.php  php.ini  wp-admin  wp-config.php  wp-content
```

Any ideas?

**Comment by bshaffer** [comment]
Hello! Yes, you need to deploy your project from inside `my-wordpress-project`. I have in the README "CD into your WordPress project directory and run the following command to deploy", which is not clear enough. I will update the README to fix this issue!

**Comment by bshaffer** [comment]
Fixed in #700
